queries.sql

```diff
@@ -24,3 +27,24 @@ SELECT * FROM animals where name <> 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals where weight_kg BETWEEN 10.4 AND 17.3;
 
+/*
+  *Queries of feature 2
+*/
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT AVG(escape_attempts) FROM animals WHERE neutered = false;
+SELECT AVG(escape_attempts) FROM animals WHERE neutered = true;
```

Comment:
- [ ] Kindly be noticed that you can use `Group By` instead of two queries.
```sql
SELECT neutered, AVG(escape_attempts) FROM animals GROUP BY neutered;
```

Reply:
Clever... and makes more sense! Thanks again!